bin/run_tests.sh

```diff
@@ -151,7 +151,7 @@ grep -A2 -B1 ")\sFailure:\|)\sError:" rails_test.log | \
     grep -v ")\sFailure:\|)\sError:\|--" >> test_summary.txt
 
 # the return code will be 1 if either the js or rails tests exited with 1
-(( RETURN_CODE = exit_status || $code ))
+(( RETURN_CODE = RETURN_CODE || $code ))
```

Comment:
Since `exit_status` wasn't actually getting defined and then `$code` gets overwritten with the exit status of the Ruby test runs, previously the JS test run results were getting ignored. Now the JS results get stored in `RETURN_CODE` so if either fails we will get the correct failed returned code